ui/src/styles/style.css

```diff
@@ -189,6 +192,10 @@ tr:hover td {
   color: red;
 }
 
+.managerNotice {
```

Comment:
Unless I missed it, `managerNotice` is an id in the html (line 119). Is there also a class somewhere?